pkg/cmd/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -97,7 +97,7 @@ func NewDefaultOffPluginsFunc(kubeDefaultOffAdmission sets.String) func() sets.S
 		kubeOff.Delete(openshiftAdmissionPluginsForKube...)
 		kubeOff.Delete(customresourcevalidationregistration.AllCustomResourceValidators...)
 		// temporarily disable RBR until we move it to a CRD (it is causing install timeout failures)
-		kubeOff.Insert("authorization.openshift.io/RestrictSubjectBindings")
+		//kubeOff.Insert("authorization.openshift.io/RestrictSubjectBindings")
```

Comment:
this is going to ripple through integration tests and you should not need to do it.